.gitignore

```diff
@@ -45,5 +46,7 @@ nebari-config.yaml
 
 .vscode/
 .pytest_cache
-
+.ipynb_checkpoints
+.DS_Store
 /.ruff_cache
+videos/
```

Comment:
Nit: can we move this up? Stuff that is very much project related should be at the top an not buried down below with the general ignores for all kinds of Python tools.